agent/provider/src/cli/keystore.rs

```diff
@@ -44,9 +47,11 @@ impl KeystoreConfig {
 
 fn list(config: ProviderConfig) -> anyhow::Result<()> {
     let cert_dir = cert_dir_path(&config)?;
-    let keystore = Keystore::load(&cert_dir)?;
-    util::print_cert_list_header();
-    keystore.visit_certs(util::print_cert_list_row)?;
+    let table_builder = CertTableBuilder::new();
+    let table_builder = util::visit_certificates(&cert_dir, table_builder)?;
+    let table = table_builder.build();
+    let output = CommandOutput::from(table);
+    output.print(false);
```

Comment:
Consider splitting `print` into two separate functions i.e. `print_json` and `print_plain` or sth like that, because passing bool directly to function `print` is unreadable.

Reply:
I noticed there is a global `--json` arg, so just added a support of it for keystore cmdline args.